spec-main/api-context-bridge-spec.ts

```diff
@@ -2,17 +2,33 @@ import { BrowserWindow, ipcMain } from 'electron/main';
 import { contextBridge } from 'electron/renderer';
 import { expect } from 'chai';
 import * as fs from 'fs-extra';
+import * as http from 'http';
 import * as os from 'os';
 import * as path from 'path';
 
 import { closeWindow } from './window-helpers';
 import { emittedOnce } from './events-helpers';
+import { AddressInfo } from 'net';
 
 const fixturesPath = path.resolve(__dirname, 'fixtures', 'api', 'context-bridge');
 
 describe('contextBridge', () => {
   let w: BrowserWindow;
   let dir: string;
+  let server: http.Server;
+
+  before(async () => {
+    server = http.createServer((req, res) => {
+      res.setHeader('Content-Type', 'text/html');
+      res.end('<html></html>');
```

Comment:
is the `<html>` necessary?

Reply:
Just keeping it consistent with what we had before.  Plus I don't like the idea of a zero-byte document 😆

Reply:
this server didn't exist before, so there's nothing to be consistent with. and zero-byte documents are just fine :)

Reply:
Updated